src/__experimental__/components/checkable-input/checkable-input.component.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { validProps } from '../../../utils/ether';
+import { StyledCheckableInput, StyledCheckableInputWrapper } from './checkable-input.style';
+import FormField from '../form-field';
+import HiddenCheckableInput from './hidden-checkable-input.component';
+import guid from '../../../utils/helpers/guid/guid';
```

Comment:
We don't need probably to have double guid here
`import guid from '../../../utils/helpers/guid'`